### ISSUE TITLE
Bad behavior when combined with router crate

### ISSUE THREAD

**Issue by heartsucker** [heartsucker/iron-csrf#16]
For example:

```rust
router! {
    index: any "index" => index,
    static: get "static" => static_,
}
```

In the browser, the request fails because the browser receives two tokens, one from `index` and one from `static_`. The latter gets sent in the cookie, and the former gets set in the form field / URL. They don't match, so the middleware rejects the request with a 403.

**Comment by heartsucker** [comment]
This is likely not directly related to the router crate. This behavior was verified with the addition of debug logging to the relevant code in the `csrf` crate. I need to look into how firefox handles cookies.

**Comment by heartsucker** [comment]
This is likely not directly related to the router crate. This behavior was verified with the addition of debug logging to the relevant code in the `csrf` crate. I need to look into how firefox handles cookies. This also happens in Chrome, so chances are I'm doing something wrong.

**Comment by heartsucker** [comment]
Workaround: have everyone browse websites via `curl`.

**Comment by heartsucker** [comment]
Fixed by a573664
